addon/components/ember-table-row.js

```diff
@@ -75,6 +76,13 @@ export default class EmberTableRow extends Component {
   }
 
   click(event) {
-    this.get('targetObject').send('onRowClicked', event, this.get('rowIndex'));
+    const tableObject = this.get('row.api.targetObject');
+    tableObject.send('onRowClicked', event, this.get('rowIndex'));
+
+    this.sendAction('onRowClicked', event, this.get('rowIndex'), this.get('rowValue'));
```

Comment:
There are 2 actions sent in this case: one for table for row selection. The other is for outer controller